Python/Product/TestAdapter.Executor/Services/ExecutorService.cs

```diff
@@ -88,17 +87,27 @@ sealed class TestRedirector : Redirector {
                 _debugSecret,
                 _debugPort.ToString(),
                 GetDebuggerSearchPath(_projectSettings.UseLegacyDebugger),
-                String.Format("--junitxml={0}", outputfile)
             };
 
-            foreach (TestCase test in tests) {
-                var executionTestPath = test.GetPropertyValue<string>(Pytest.Constants.PytestIdProperty, default);
-                if (String.IsNullOrEmpty(executionTestPath)) {
-                    Debug.WriteLine("Pytest execution path missing for testcase {0}", test.FullyQualifiedName);
-                    continue;
+            // For a small set of tests, we'll pass them on the command
```

Comment:
Do we care about deleting the temp testListFilePath?

its nicer for debugging not to delete it

Reply:
I don't mind it